etna/analysis/outliers/median_outliers.py

```diff
@@ -36,7 +36,7 @@ def get_anomalies_median(
     outliers_per_segment = {}
     segments = ts.segments
     for seg in segments:
-        anomalies = []
+        anomalies: typing.List[int] = []
```

Comment:
`from typing import List`?

Reply:
In this file we are already using `typing` as is without importing anything. We can rewrite it if necessary.